forge/test/helpers/PoolSetupHelper.t.sol

```diff
@@ -0,0 +1,144 @@
+    address constant ADDR_2 = 0x15d34AAf54267DB7D7c367839AAf71A00a2C6A65;
+    address constant ADDR_3 = 0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc;
+    address constant ADDR_4 = 0x976EA74026E726554dB657fA54763abd0C3a0aa9;
+
+    L2Encoder l2Encoder;
+    TestToken token;
+    TestChainlinkOracle chainlinkOracle;
+    TestChainlinkOracle ethOracle;
+    ChainlinkOracleWrapper oracleWrapper;
+    ChainlinkOracleWrapper settlementEthOracle;
+    PoolFactory poolFactory;
+    InvariantCheck invariantCheck;
+    PoolKeeper poolKeeper;
+    KeeperRewards keeperRewards;
+    AutoClaim autoclaim;
+
+    LeveragedPool leveragedPool;
+    PoolToken long;
+    PoolToken short;
+    PoolCommitter poolCommiter;
+
+    function _deployPoolSetupContracts() private {
+        l2Encoder = new L2Encoder();
+
+        // Deploy test ERC20 token
+        token = new TestToken("TEST TOKEN", "TST1");
+        token.mint(DEPLOYER, DEFAULT_MINT_AMOUNT);
+
+        // Deploy test oracle
+        ethOracle = new TestChainlinkOracle();
+        ethOracle.setPrice(3000 * 1e8);
+
+        chainlinkOracle = new TestChainlinkOracle();
+        oracleWrapper = new ChainlinkOracleWrapper(address(chainlinkOracle), DEPLOYER);
+
+        // keeper oracle
+        settlementEthOracle = new ChainlinkOracleWrapper(address(ethOracle), DEPLOYER);
+
+        // Deploy and init pool
+        poolFactory = new PoolFactory(FEE_RECEIVER, GOVERNANCE);
+
+        invariantCheck = new InvariantCheck(address(poolFactory));
+        poolFactory.setInvariantCheck(address(invariantCheck));
+
+        poolKeeper = new PoolKeeper(address(poolFactory));
+        poolFactory.setPoolKeeper(address(poolKeeper));
+        poolFactory.setFee(DEFAULT_FEE);
+
+        keeperRewards = new KeeperRewards(address(poolKeeper));
+        poolKeeper.setKeeperRewards(address(keeperRewards));
+
+        autoclaim = new AutoClaim(address(poolFactory));
+        poolFactory.setAutoClaim(address(autoclaim));
+    }
+
+    /**
+     * Deploys a new instance of a pool, as well as an ERC20 token to use as a settlement token.
+     * @param poolCode The pool identifier
+     * @param frontRunningInterval The front running interval value. Must be less than the update interval
+     * @param updateInterval The update interval value
+     * @param leverage The amount of leverage the pool will apply
+     * @param feeAddress The address to transfer fees to on a fund movement
+     * @param fee The fund movement fee.
+     * @param mintFee Mint fee
+     * @param burnFee Burn fee
+     * @param changeInterval Change interval
+     */
+    function setupPoolAndTokenContracts(
+        string memory poolCode,
+        uint32 frontRunningInterval,
+        uint32 updateInterval,
+        uint16 leverage,
+        address feeAddress,
+        uint256 fee,
+        uint256 mintFee,
+        uint256 burnFee,
+        uint256 changeInterval
+    ) internal {
+        skip(100000); // start block.timestamp @ 100000
```

Comment:
Why?

Reply:
Arbitrary value because`block.timestamp` starts @ 0, which is unrealistic.